src/configuration/app-containers.md

```diff
@@ -386,21 +387,21 @@ The `hooks` let you define shell commands to run during the build and deployment
 They can be executed at various points in the lifecycle of the
 application (build/deploy).
 
-Possible hooks are:
+Possible hooks are below:
 
 -   **build**: We run a build hook before your application has been
     packaged. No other services are accessible at this time since the
     application has not been deployed yet.
 -   **deploy**: We run a deploy hook after your application has been
     deployed and started. You can access other services at this stage
-    (MySQL, Solr, Redis...). However, the disk where the application lives is read-only at this point.
+    (MySQL, Solr, Redis, etc.), however, the disk where the application lives is read-only at this point.
```

Comment:
Why join the sentences?  It seems fine as its own sentence.  (The "etc" is a good change, though.)